website/docs/language/state/purpose.mdx

```diff
@@ -35,8 +35,8 @@ support tags.
 Therefore, for mapping configuration to resources in the real world,
 Terraform uses its own state structure.
 
-Terraform expects that each remote object is bound to only one resource
-instance, which is normally guaranteed by Terraform being responsible for
+Terraform expects that each remote object on the provider is bound to only one resource
+instance in the configuration, which is normally guaranteed since Terraform is responsible for
```

Comment:
"guaranteed by" reads more clearly to me than "guaranteed since." I would not change this clause.

Reply:
@crw  - Thank you. Could we change this to - "which is normally guaranteed by Terraform since Terraform is responsible for"

Reply:
I think that is OK. @laurapacilio, please let us know if that sounds grammatically correct.

Reply:
Hi both - apologies for the delay here! I want to think about this sentence in context (also I don't love how the original text separates out lines that are clearly related to one another into separate paragraphs). So I pulled out some additional sentences that come after this and rewrote all of them a bit. The goal is to make sure it all flows together. What do you think of rewriting it to be something like what I have below?

"Terraform expects that each remote object is bound to only one resource instance in the configuration. If a remote object is bound to multiple resource instances, the mapping from configuration to the remote object state becomes ambiguous, and Terraform may behave unexpectedly. Terraform can guarantee a one-to-one mapping when it creates objects and records their identities in the state. When importing objects created outside of Terraform, you must make sure that each distinct object is imported to only one resource instance."

Reply:
Hi @laurapacilio - The paragraph looks more comprehensive. A minor correction in the paragraph is needed, though.
@crw  - please suggest.
"Terraform expects that each remote object is bound to only one resource instance in the configuration. If a remote object is bound to multiple resource instances, the mapping from configuration to the remote object in the state becomes ambiguous, and Terraform may behave unexpectedly. Terraform can guarantee a one-to-one mapping when it creates objects and records their identities in the state. When importing objects created outside of Terraform, you must make sure that each distinct object is imported to only one resource instance."

Reply:
That minor change looks good (took me a while to find it).

Reply:
Thank you @crw . 
@laurapacilio - All good . Below is the copy of your paragraph which includes my change that @crw  has consented to.
"Terraform expects that each remote object is bound to only one resource instance in the configuration. If a remote object is bound to multiple resource instances, the mapping from configuration to the remote object in the state becomes ambiguous, and Terraform may behave unexpectedly. Terraform can guarantee a one-to-one mapping when it creates objects and records their identities in the state. When importing objects created outside of Terraform, you must make sure that each distinct object is imported to only one resource instance."

Reply:
This looks great! Could you please update the document to add this new, combined paragraph?  Once you do that, I feel comfortable approving and merging :)

Reply:
@laurapacilio  - I have updated the existing document with the new, combined paragraph.